schema/vrs.yaml

```diff
@@ -532,9 +532,10 @@ definitions:
 
   Gene:
     description: >-
-      A reference to an external gene system, used as a location for
-      variation.  Currently, the `ncbigene` namespace is required. See
-      https://registry.identifiers.org/registry/ncbigene.
+      A gene is an authoritative representation of one or more heritable
```

Comment:
I prefer the first sentence of the original definition. We don't actually care what the definition is; instead, we care that someone else defined it. I'm ambivalent about the ncbigene part.

Reply:
If we revert then we should remove the `ncbigene` requirement comment. We could leave it as an example. In reality, we should be pushing for HGNC as that is the standard being promoted by HL7 FHIR guidance even though it too is not strictly required.

Reply:
Reverted.